addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -118,7 +118,7 @@ export class ModelFieldSelectorPopover extends Component {
         update: Function,
     };
     static defaultProps = {
-        filter: (value) => value.searchable && value.type != "json",
+        filter: (value) => value.searchable && value.type != "json" && value.type !== "separator",
```

Comment:
This commit's content is LGTM, just wondering if we did not have this feature prior to 18.3? separator and search / group by?